platform/docs/docs/migration-guide.md

```diff
@@ -0,0 +1,492 @@
+---
+sidebar_position: 10
+sidebar_label: Migration Guide
+---
+
+# Migration Guide
+
+On this page, we will provide a guide to migrating from OHIF v2 to v3. Please note
+that this document is a work in progress and will be updated as we move forward.
+
+
+## Summary of Changes
+
+OHIF v3 is a complete re-architecture of the OHIF v2 to make it more modular and
+easier to maintain. The main differences are:
+
+- Extensions don't inject their modules into the viewer, they will make them available
+  to be used by the viewer.
+- To use the modules provided by the extensions, you need to write a [Modes](./platform/modes/index.md). Modes
+are configuration objects that will be used by the viewer to load the modules.
+- App configuration structure is different, mainly the `servers` is renamed to `dataSources`.
```

Comment:
- Apps can be customized significantly more than previously by providing configuration code int he customizationModule section.